### PR TITLE
#439 MainRemote should allow additional arguments to be passed

### PR DESCRIPTION
**Pull request by HDouss**
#439 MainRemote modification to allow additional arguments and its unit test

### CODE REVIEW

**Comment by davvd** [comment]
@hdouss Thanks, someone will review your pull request soon

**Comment by davvd** [comment]
@dmzaytsev please review, thanks

**Comment by dmzaytsev** [comment]
@rultor merge

File: src/test/java/org/takes/http/MainRemoteTest.java [hunk 1]
```diff
@@ -62,6 +62,27 @@ public void exec(final URI home) throws IOException {
     }
 
     /**
+     * MainRemote passes additional arguments.
+     * @throws Exception If some problem inside
+     */
+    @Test
+    public void passesArgumentsToApp() throws Exception {
+        final String[] addArgs = {"works well!"};
```
Comment (yegor256) [inline src/test/java/org/takes/http/MainRemoteTest.java]: @HDouss just `args` won't be enough? http://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html

File: src/main/java/org/takes/http/MainRemote.java [hunk 2]
```diff
@@ -53,11 +54,26 @@
     private final transient Class<?> app;
 
     /**
+     * Additional arguments to be passed to the main class.
+     */
+    private final transient String[] addargs;
```
Comment (yegor256) [inline src/main/java/org/takes/http/MainRemote.java]: @HDouss just `args` is not enough? http://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html

**Comment by yegor256** [comment]
@HDouss two comments from me

**Comment by HDouss** [comment]
@yegor256 done !

**Comment by yegor256** [comment]
@rultor try to merge

**Comment by davvd** [comment]
@ypshenychka please, help us to evaluate quality here, as per [par.24](http://at.teamed.io/policy.html#24)

**Comment by davvd** [comment]
@rultor please deploy

**Comment by ypshenychka** [comment]
@dmzaytsev 
According to our [QA Rules](http://www.teamed.io/qa.html):
>The code reviewer found at least three problems in the code.

No issues were found during code review. 
Please confirm that you'll try to find at least three problems while future reviews.

**Comment by dmzaytsev** [comment]
@ypshenychka I confirm

**Comment by ypshenychka** [comment]
@dmzaytsev thanks

**Comment by ypshenychka** [comment]
@davvd Quality is acceptable.

**Comment by davvd** [comment]
> @davvd Quality is acceptable.

@ypshenychka thanks, got it. everybody, please try to make it good next time

**Comment by davvd** [comment]
@dmzaytsev paied 10 mins to @ypshenychka for QA review (payment ID is `71818766`)... Much appreciated! **17 mins** was added to your account, payment ID `71818775`, time spent is 16 hours and 1 min... extra minutes for review comments (c=2)... +17 to your rating, your total score is [+970](http://www.netbout.com/b/35166?open=rating)

**Pull request merged**